examples/keyvalue/Makefile

```diff
@@ -7,4 +9,7 @@ server/server: $(wildcard server/*.go) kv/types.go
 .PHONY: all
 all: server/server
 
-.DEFAULT_GOAL: all
+.PHONY: deps
+deps:
+	$(ECHO_V)go get -v go.uber.org/thriftrw
```

Comment:
It is now, but what happens when 2.0 comes out and we're using 1.0 in glide, but this fetches the latest (2.0)?